src/CustomerProfile.js

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from "react";
+
+const CustomerProfile = (props) => {
+  const [customer, setCustomer] = useState("");
+
+  let id = props.id;
+
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/customers/${id}`)
+      .then((response) => response.json())
+      .then((data) => setCustomer(data))
+      .catch((err) => console.log(err));
+  }, [id]);
+  if (customer.id) {
```

Comment:
Best to try and leave a space between these, so its easier to seperate whats going on 😄